cmd/kubeadm/app/cmd/phases/util.go

```diff
@@ -17,8 +17,9 @@ limitations under the License.
 package phases
 
 import (
-	"k8s.io/component-base/version"
+	"os"
 
+	"k8s.io/component-base/version"
```

Comment:
please separate these:
```
os

k8s.io/component-base/version"

kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
```

as mentioned here:
https://github.com/kubernetes/kubernetes/pull/108410#discussion_r817159911

Reply:
So sorry, I though only need to separate  stdlib / k8s, Updated.